charts/k6-operator/values.yaml

```diff
@@ -15,158 +24,301 @@ global:
     pullSecrets: []
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # customAnnotations -- Custom Annotations to be applied on all resources
 customAnnotations: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # podAnnotations -- Custom Annotations to be applied on all pods
 podAnnotations: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # customLabels -- Custom Label to be applied on all resources
 customLabels: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # podLabels -- Custom Label to be applied on all pods
 podLabels: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # nodeSelector -- Node Selector to be applied on all containers
 nodeSelector: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
 # @schema
 # affinity -- Affinity to be applied on all containers
 affinity: {}
 
 # @schema
+# additionalProperties: true
 # required: false
 # type: object
```

Comment:
```suggestion
# type: array
```
I'm getting the following error now:
```
- tolerations: Invalid type. Expected: object, given: array
```
And tolerations are normally defined as an array:
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

Reply:
It appears this change request was done in a new PR by another contributor over here:
https://github.com/grafana/k6-operator/pull/481/files